geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSet.java

```diff
@@ -359,8 +359,10 @@ public int scard() {
   }
 
   @Override
-  public void applyAddByteArrayDelta(byte[] bytes) {
-    membersAdd(bytes);
+  public void applyAddByteArrayDelta(List<byte[]> byteArrays) {
```

Comment:
this should be named `applyAddByteArraysDelta`

Reply:
Mooted after rebase